src/lib/ReactDataTable/ReactDataTableProps.ts

```diff
@@ -56,6 +56,11 @@ export interface ReactDataTableProps<TData> {
    */
   showPaging?: boolean;
 
+  /**
+   * boolean flag to hide the possibility to change the page size
+   */
+  withoutPageSizeChange?: boolean;
```

Comment:
@neotob 

Maybe `hidePageSizeChange` would be a better name?

Reply:
changed 👍